src/commands/deployment/onboard.ts

```diff
@@ -109,6 +111,7 @@ export const setConfiguration = (
     fs.writeFileSync(defaultConfigFile(), jsonData);
     return true;
   } catch (err) {
+    // TOFIX: write some comments on why this error can be ignore
     logger.error(
       `Unable to set storage account and table names in configuration file. \n ${err}`
```

Comment:
Is there a reason this error can be ignored?

Reply:
i just added a TOFIX on why this error can be ignored. It is ignored. I do not change anything. Can you elaborate on your question? thanks